src/pseudopeople/configuration/interface.py

```diff
@@ -0,0 +1,52 @@
+"""
+===========================
+The Configuration Interface
+===========================
+
+An interface for user's to interact with the configuration (config) in pseudopeople.
+
+In ``pseudopeople`` a configuration is used to provide noise level that get passed to the noise functions to apply
+various types of noise to given datasets.  Users can provide their own configuration that will override the deault
+values.
+.. code-block:: python
+>>> import pseudopeople as pp
+>>> pp.get_config("decennial_census")
+>>> user_config = {"decennial_census": {"row_noise": {"probability": 0.1},},}
```

Comment:
This is not a valid configuration. You're missing the layer for the noise type (i.e. "omit_row").